data/hai/hai reveal 0 prologue.txt

```diff
@@ -0,0 +1,736 @@
+	source "Hai-home"
+	waypoint "Cardax"
+	waypoint "Hassaleh"
+	waypoint "Moktar"
+	waypoint "Oblate"
+	waypoint "Rajak"
+	waypoint "Rigel"
+	waypoint "Sumar"
+	waypoint "Ultima Thule"
+	waypoint "Volax"
+	to offer
+		has "Hai Reveal: Meet the Team: Alondo: done"
+		has "Hai Reveal: Meet the Team: Xilin: done"
+		has "Hai Reveal: Meet the Team: Samuel: done"
+	on offer
+		conversation
+			`When you exit onto the landing pads, you find that Terry has just finished ensuring your ship is refueled. "Alright, Captain. I have a simple mission for you. We need someone to scout out the ship frequencies in various uninhabited systems in the Far North. We know how many merchants are arriving here, but we're not sure how many are attempting and failing, for one reason or another. I need you to fly through these systems and report back here with what you find."`
+			`	She takes a deep breath before saying: "<waypoints>. Think you can remember all those? Good." She doesn't even wait for a response. "Stick around in each system if you can, as I want to know just how many merchants, pirates, and Navy ships are passing through that region."`
+			`	She tosses you a data pad containing coordinates to all the systems she mentioned and starts walking back inside. "Just come back here once you're finished."`
+				accept
+	on visit
+		dialog phrase "generic waypoint on visit"
+
+
+
+mission "Hai Reveal: Meet the Team: Sayari"
+	landing
+	name "Important Message"
+	description "Deliver an important message from the Hai to the Republic on <stopovers>. Return to <destination> when finished."
+	source "Hai-home"
+	stopover "Earth"
+	to offer
+		has "Hai Reveal: Meet the Team: Terry: done"
+	on offer
+		conversation
+			`Terry quickly takes a copy of your ship's system logs upon your return. "Thank you, Captain. I'll have these analyzed in no time." Terry runs back into the building just as Sayari comes out carrying a small box.`
+			`	"Captain <last>. I know we are asking much of you, but I have a task of my own." She hands you the box, which displays official Hai government branding. Inside is an encrypted data chip and a metallic badge, with a symbol minutely engraved into the front that must be the Hai government's emblem. Surrounding it in a circle is some sort of pattern, which reminds you of the codes found on identification cards and licenses in human space. On the other side, there is extensive writing in a script that looks to be Hai. From this, you presume that the badge is some sort of diplomatic ID.`
+			`	"Please deliver this message to the Republic. I trust that they will disseminate it to your other governments. This badge should make it easier if they give you trouble."`
+			choice
+				`	"Consider it done."`
+					accept
+				`	"Various? We only have two governments."`
+			`	"Oh," she says with a smirk, "we've treated the Syndicate as a de facto government for many decades now; with the amount of influence they have it was the only prudent thing to do."`
```

Comment:
Sayari doesn't say "various", though?

More generally, I feel like Sayari's response to this is a little awkward, like she's more smug that the player is wrong than honestly telling them why the Hai consider the Syndicate as a separate government.

Reply:
The "various" is a holdover from the previous introduction, as is Sayari's response. But I'm now realizing that this doesn't really line up with my plans for the Syndicate, so this whole but can just be tossed.